src/mlpack/methods/preprocess/load_save_image_main.cpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file load_save_image_main.cpp
+ * @author Jeffin Sam
+ *
+ * A CLI executable to load and image dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core.hpp>
+
+using namespace mlpack;
+using namespace mlpack::util;
+using namespace arma;
+using namespace std;
+using namespace mlpack::data;
```

Comment:
If you use ``data::ImageInfo`` below, this isn't necessary.

Reply:
It is required for line 67 and 68 , also I cannot change it.

Reply:
You mean ``PARAM_FLAG("save", "Save a dataset as images", "s");``?

Reply:
Sorry 71 and 72 , 
```
PARAM_MODEL_IN(ImageInfo, "input_model", "Input Image Info model.", "m");
PARAM_MODEL_OUT(ImageInfo, "output_model", "Output Image Info model.", "M");
```

I can't pass data::Imageinfo , throws some error